### PR TITLE
6122 admin base url

### PR DESCRIPTION
**Pull request by alibombaye**
updated to always route via /admin

### CODE REVIEW

File: app/pages/adduser/confirmation/ui.test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { extractInnerText } from '../../../test-utils/helper';
 import { organisationsApiLocalhost } from '../../../test-utils/config';
 import organisationDetails from '../../../test-utils/fixtures/organisationDetails.json';
 
-const path = 'http://localhost:1234/organisations/org1/adduser/confirmation?id=user1';
+const path = 'http://localhost:1234/admin/organisations/org1/adduser/confirmation?id=user1';
```
Comment (BenBJSS) [inline app/pages/adduser/confirmation/ui.test.js:8]: Do we need to put baseUrl in here?

Comment (alibombaye) [reply app/pages/adduser/confirmation/ui.test.js:8]: I've decided to hard code it in the ui.test to make it explicit this is what the page we are testing is

File: testcafeRunner.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-import { env } from './app/config';
+import { env, baseUrl } from './app/config';
 import { FakeAuthProvider } from './app/test-utils/FakeAuthProvider';
 
 let testcafe;
 
 const authProvider = new FakeAuthProvider();
 const app = new App(authProvider).createApp();
-app.use('/', routes(authProvider));
+app.use(baseUrl || '/', routes(authProvider));
```
Comment (kat1906) [inline testcafeRunner.js:11]: don't think we need the or if we have default in config

File: testcafeRunner.js [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+if (baseUrl) {
```
Comment (kat1906) [inline testcafeRunner.js:12]: what is this for?

**Pull request merged**